build/yaml/botbuilder-js-daily.yml

```diff
@@ -0,0 +1,266 @@
+#
+# This runs under BotBuilder-JS-daily-yaml.
+#
+
+# "name" here defines the build number format. Build number is accessed via $(Build.BuildNumber)
+name: $(Build.BuildId)
+
+pool:
+  vmImage: 'macOS-10.15'
+
+variables:
+  NodeVersion: 10.x
+#  PackageVersion: Define this in Azure to be settable at queue time. Ex: 4.11.0-dev.{DateStamp}.{CommitHash} 
+#  PreviewPackageVersion: Define this in Azure to be settable at queue time. Ex: 4.11.0-dev.{DateStamp}.{CommitHash}
+
+steps:
+- powershell: |
+   # Replace {DateStamp} and {CommitHash} tokens with the actual values in vars PackageVersion and PreviewPackageVersion
+   $dateStamp = (Get-Date -format "yyyyMMdd");
+   $commitHash = "$(Build.SourceVersion)".SubString(0,7);
+   
+   "Raw PackageVersion = $(PackageVersion)";
+   $v = "$(PackageVersion)".Replace("{DateStamp}",$dateStamp).Replace("{CommitHash}",$commitHash);
+   Write-Host "##vso[task.setvariable variable=PackageVersion;]$v";
+   "Resolved PackageVersion = $v";
+   
+   "Raw PreviewPackageVersion = $(PreviewPackageVersion)";
+   $ppv = "$(PreviewPackageVersion)".Replace("{DateStamp}",$dateStamp).Replace("{CommitHash}",$commitHash);
+   Write-Host "##vso[task.setvariable variable=PreviewPackageVersion;]$ppv";
+   "Resolved PreviewPackageVersion = $ppv";
+  displayName: 'Resolve package version variables'
+
+- template: js-build-steps.yml
+
+- task: Npm@1
+  displayName: 'npm run eslint'
+  inputs:
+    command: custom
+    verbose: false
+    customCommand: 'run eslint'
+
+- task: Npm@1
+  displayName: 'npm pack adaptive-expressions'
+  inputs:
+    command: custom
+    workingDir: 'libraries/adaptive-expressions'
+    verbose: false
+    customCommand: pack
+    publishEndpoint: 'MyGet botbuilder JS Scratch Feed'
+
+- task: Npm@1
+  displayName: 'npm pack botframework-streaming'
+  inputs:
+    command: custom
+    workingDir: 'libraries/botframework-streaming'
+    verbose: false
+    customCommand: pack
+    publishEndpoint: 'MyGet botbuilder JS Scratch Feed'
+
+- task: Npm@1
+  displayName: 'npm pack botbuilder'
+  inputs:
+    command: custom
+    workingDir: libraries/botbuilder
+    verbose: false
+    customCommand: pack
+    publishEndpoint: 'MyGet botbuilder JS Scratch Feed'
```

Comment:
Can you sort these `packs` to match the order in `libraries/`?

Reply:
Good idea. Put in Windows folders order.